<ui version="4.0" >
 <class>HBondSettingsWidget</class>
 <widget class="QWidget" name="HBondSettingsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>233</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="labelWidth" >
     <property name="text" >
      <string>Width:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSlider" name="widthSlider" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>3</number>
     </property>
     <property name="singleStep" >
      <number>1</number>
     </property>
     <property name="pageStep" >
      <number>2</number>
     </property>
     <property name="value" >
      <number>2</number>
     </property>
     <property name="sliderPosition" >
      <number>2</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance" >
      <bool>false</bool>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval" >
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Cut-off radius:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QDoubleSpinBox" name="radiusSpin" >
     <property name="suffix" >
      <string>A</string>
     </property>
     <property name="value" >
      <double>2.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QDoubleSpinBox" name="angleSpin" >
     <property name="suffix" >
      <string>Â°</string>
     </property>
     <property name="minimum" >
      <double>1.000000000000000</double>
     </property>
     <property name="maximum" >
      <double>180.000000000000000</double>
     </property>
     <property name="value" >
      <double>120.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Cut-off angle:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
